tests/test_queries.py

```diff
@@ -0,0 +1,183 @@
+import pytest
+from indra_cogex.client.queries import *
+from indra.statements import *
+from indra_cogex.representation import Node
+
+from .test_neo4j_client import _get_client
+
+
+@pytest.mark.nonpublic
+def test_get_genes_in_tissue():
+    # Single query
+    client = _get_client()
+    genes = get_genes_in_tissue(client, ("UBERON", "UBERON:0002349"))
+    assert len(genes) == 10213
+    node0 = genes[0]
+    assert isinstance(node0, Node)
+    assert node0.db_ns == 'HGNC'
+
+
+@pytest.mark.nonpublic
+def test_get_tissues_for_gene():
+    # Single query
+    client = _get_client()
+    tissues = get_tissues_for_gene(client, ("HGNC", "9896"))
+    assert len(tissues) == 274
+    node0 = tissues[0]
+    assert isinstance(node0, Node)
+    assert node0.db_ns == 'UBERON'
+
+
+@pytest.mark.nonpublic
+def test_is_gene_in_tissue():
+    client = _get_client()
+    gene = ("HGNC", "9896")  # RBM10
+    tissue = ("UBERON", "UBERON:0035841")  # esophagogastric junction muscularis propria
+    assert is_gene_in_tissue(client, gene, tissue)
+
+
+@pytest.mark.nonpublic
+def test_get_go_terms_for_gene():
+    client = _get_client()
+    gene = ("HGNC", "16813")  # PPP1R27
+    go_terms = get_go_terms_for_gene(client, gene)
+    assert len(go_terms) == 2
+    node0 = go_terms[0]
+    assert isinstance(node0, Node)
+    assert node0.db_ns == 'GO'
+
+
+@pytest.mark.nonpublic
+def test_get_genes_for_go_term():
+    # Single query
+    client = _get_client()
+    go_term = ("GO", "GO:0000978")
+    genes = get_genes_for_go_term(client, go_term)
+    assert len(genes) == 1146
```

Comment:
I'd be careful to hard-code numbers in this (and all of the other) tests here, since these might change if the GO Annotations database or other underlying data sources get updated. It's much less likely that a single element will be removed, so perhaps check for something specific?

Reply:
Good point, will fix